HelloWars/Arena/Pages/Converters/BoolToOpacityConverter.cs

```
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Windows.Data;
+
+namespace Arena.Pages.Converters
+{
+    public sealed class BoolToOpacityConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            return (value is bool && (bool)value) ? 1 : 0.4;
```

Do we want to throw an exception if not boolean?